### DISCUSSION TITLE
Turn off streaming replay

### DISCUSSION THREAD

**Discussion by anne-markis** [jsforce/jsforce#1447] [Q&A]
It seems that streaming API replay is enabled by default. Is there a way to disable this?

**Comment by ronwestbrook** [comment]
I am actually trying to make use of replay, but I haven't been able to confirm or deny that it is working.  If it does work, is there something I need to add to my code to automatically process missed messages at the time the application comes back online?

**Comment by yaronyagoda** [comment]
I am also looking for a way to reply events in JS and I can't seem to fins it anywhere...

**Comment by internalfx** [comment]
I could not find anything in the docs myself, although issue #814 certainly alludes that it is possible to control replay in the streaming client.

@mattdrees You seem to know something about this? Can you confirm if replay works, can it be configured?
